stdlib/LinearAlgebra/test/dense.jl

```diff
@@ -22,10 +22,10 @@ Random.seed!(1234323)
     @testset "for $elty" for elty in (Float32, Float64, ComplexF32, ComplexF64)
         ainit = convert(Matrix{elty}, ainit)
         for a in (copy(ainit), view(ainit, 1:n, 1:n))
-            @test cond(a,1) ≈ 122.15725126320953 atol=0.5
-            @test cond(a,2) ≈ 78.44837047684149 atol=0.5
-            @test cond(a,Inf) ≈ 174.10761543202744 atol=0.4
-            @test cond(a[:,1:5]) ≈ 6.7492840150789135 atol=0.01
+            @test cond(a,1) ≈ 198.3324294531168 atol=0.5
```

Comment:
These tests are the worst...

Reply:
Seriously. What a badly designed test...